src/components/Status.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const Status = ({ status }) => {
+    if(status === 'Login error!') {
```

Comment:
arata f hacky sa testezi un string fix :)

Reply:
- Refactored Status component under a new name Message component

- Used classnames package to switch between error red msg and normal message using a new redux boolean flas isStatusError and removed the fix string check